core/templates/dev/head/pages/topics_and_skills_dashboard/TopicsAndSkillsDashboard.js

```diff
@@ -19,27 +19,30 @@ oppia.constant(
   'EDITABLE_TOPIC_DATA_URL_TEMPLATE', '/topic_editor_handler/data/<topic_id>');
 
 oppia.constant('EVENT_TOPIC_DELETED', 'topicDeleted');
+oppia.constant('CAN_USER_CREATE_TOPIC', 'canUserCreateTopic');
 
 oppia.controller('TopicsAndSkillsDashboard', [
   '$scope', '$rootScope', '$http', '$window',
   'AlertsService', 'TopicsAndSkillsDashboardBackendApiService',
   'UrlInterpolationService', 'TopicCreationService',
-  'FATAL_ERROR_CODES', 'EVENT_TOPIC_DELETED',
+  'FATAL_ERROR_CODES', 'EVENT_TOPIC_DELETED', 'CAN_USER_CREATE_TOPIC',
   function(
       $scope, $rootScope, $http, $window,
       AlertsService, TopicsAndSkillsDashboardBackendApiService,
       UrlInterpolationService, TopicCreationService,
-      FATAL_ERROR_CODES, EVENT_TOPIC_DELETED) {
+      FATAL_ERROR_CODES, EVENT_TOPIC_DELETED, CAN_USER_CREATE_TOPIC) {
     $scope.TAB_NAME_TOPICS = 'topics';
     $scope.TAB_NAME_SKILLS = 'skills';
-    $scope.canCreateTopic = GLOBALS.canCreateTopic;
 
     TopicsAndSkillsDashboardBackendApiService.fetchDashboardData().then(
       function(response) {
         $scope.topicSummaries = response.data.topic_summary_dicts;
         $scope.skillSummaries = response.data.skill_summary_dicts;
         $scope.activeTab = $scope.TAB_NAME_TOPICS;
-        $scope.canDeleteTopic = response.data.can_delete_topic;
+        $scope.userCanCreateNewTopic = response.data.can_create_topic;
+        $rootScope.$broadcast(
+          CAN_USER_CREATE_TOPIC, $scope.userCanCreateNewTopic);
```

Comment:
This is a weird event name.

Do something like EVENT_TYPE_TOPIC_CREATION_ENABLED instead (and put this in an if condition rather than passing an arg). If you default to topic creation being disabled at the outset, I think this should achieve what you want.

Reply:
Done.